osmosis-1/osmosis.zone_assets.json

```diff
@@ -3914,6 +3917,42 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-05-08T18:00:00Z",
       "_comment": "$WEIRD Friends NFTs"
+    },
+    {
+      "chain_name": "quicksilver",
+      "base_denom": "uqjuno",
+      "path": "transfer/channel-522/uqjuno",
+      "osmosis_verified": true,
```

Comment:
Please always set to false upon initial listing. We can update to true after, but we have to validate everything first.